datalad/tests/utils.py

```diff
@@ -147,23 +213,22 @@ def skip_if_scrapy_without_selector():
         import scrapy
         from scrapy.selector import Selector
     except ImportError:
-        from nose import SkipTest
-        raise SkipTest(
+        pytest.skip(
```

Comment:
me curious: couldn't we just `from unittest import SkipTest` and leave all those `raise SkipTest` intact?  or there are some advantages from explicit `pytest.skip`?

Reply:
@yarikoptic Well, using `pytest.skip` is more `pytest`-thonic, and I assume we'll want to eventually switch all `SkipTest`s over to `pytest.skip` at some point.  Does it matter whether we keep using `SkipTest` for now?

Reply:
Not really. Just that having a minimal diff could help to orient/avoid conflicts etc. More pytest-ification could be done later